build-logic/src/main/java/qsl/internal/Versions.java

```diff
@@ -21,7 +21,7 @@ public final class Versions {
 	/**
 	 * The QSL version
 	 */
-	public static final String QSL_VERSION = "2.0.0-beta.15";
+	public static final String QSL_VERSION = "2.0.0-beta.16";
```

Comment:
I was gonna complain about the changes on Versions *but* they were actually done on upstream instead; oh god, what did you do for the merge

Reply:
:iea: I just let IntelliJ do the heavy lifting, I have no idea why the merges happened that way